.travis.yml

```diff
@@ -1,12 +1,13 @@
-dist: trusty
+dist: jammy
 language: java
 sudo: required
 jdk:
-  - oraclejdk8
-  - oraclejdk9
-  - openjdk8
-  - openjdk10
-  - openjdk11
+  - openjdk17
+  - openjdk18
```

Comment:
@scottf can you help here to get the other JDKs to work? I also couldn't find any list for travis JDK names :/

Reply:
@YunaBraska This is fixed with Jammy dist for Travis JDKs. You can see latest build succeeded for all JDKS from Java17 to Java22.